CHANGES.md

```diff
@@ -1,3 +1,6 @@
+## Version 3.0.1 (in development) 
+* Fixed bug that would cause that no local zarr datasets could be opened. 
```

Comment:
Please describe the change for the interested user. A "local zarr dataset" ist not what user see. They cannot open cached datasets from the local file store after they have downloaded data from ODP store.